jobs/sql/derivations/hiv_patient_summary_status.sql

```diff
@@ -0,0 +1,348 @@
+CREATE TABLE hiv_patient_summary_status
```

Comment:
This replaces the separate schema + extraction job that were removed, since we should be able to do this all in one go against the warehouse DB, which is simpler to maintain and should be more reliable to execute.

Reply:
same comment as above here